heron/uploaders/src/java/BUILD

```diff
@@ -107,3 +107,22 @@ genrule(
     outs = ["heron-scp-uploader.jar"],
```

Comment:
can you write a page documentation similar to other uploaders so that we have information about how to use this?